fem/intrules.hpp

```diff
@@ -27,7 +27,7 @@ class IntegrationPoint
 public:
    double x, y, z, weight;
 
-   void Init() { x = y = z = weight = 0.0; }
+   IntegrationPoint() : x(0.0), y(0.0), z(0.0), weight(0.0) { }
```

Comment:
The default constructor was intentionally left implicitly defined - that way both default-initialization and value-initialization can be used, with different behavior - similar to any fundamental type.

Is there a reason, you always want zero-initialization?

Reply:
ok, i'll revert these changes